tests/playwright/sanity/container.test.js

```diff
@@ -526,6 +528,31 @@ test.describe( 'Container tests', () => {
 		await editor.togglePreviewMode();
 		await expect( resizers ).toHaveCount( 0 );
 	} );
+
+	test( 'Test grid container gap', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost();
+
+		// Add container.
+		await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Close Navigator
+		await editor.closeNavigatorIfOpen();
+
+		// Set container type to grid.
```

Comment:
We have editor.setSelectControlValuefunction ( in pro ) maybe copy it to core if we don't have a similar one